### DISCUSSION TITLE
06.08.2021

### DISCUSSION THREAD

**Discussion by farag2** [farag2/Sophia-Script-for-Windows#225] [General]
# Before running the script read [Before running](https://github.com/farag2/Sophia-Script-for-Windows#fire-before-running-fire) & [How to use](https://github.com/farag2/Sophia-Script-for-Windows#how-to-use)

[![YT](https://img.youtube.com/vi/f529ucAipI8/0.jpg)](https://youtu.be/f529ucAipI8) [![YT](https://img.youtube.com/vi/MiQ85tVXQQA/0.jpg)](https://youtu.be/MiQ85tVXQQA)

***

[![ko-fi](https://www.ko-fi.com/img/githubbutton_sm.svg)](https://ko-fi.com/Q5Q51QUJC)

<a href="https://yoomoney.ru/to/4100116615568835"><img src="https://yoomoney.ru/i/shop/iomoney_logo_color_example.png" width=220px height=46px></a>

***

[![actions](https://img.shields.io/badge/Sophia%20News-Telegram-blue?style=flat&logo=Telegram)](https://t.me/SophiaNews) [![actions](https://img.shields.io/badge/Sophia%20Chat-Telegram-blue?style=flat&logo=Telegram)](https://t.me/Sophia_Chat) [![actions](https://img.shields.io/github/workflow/status/farag2/Sophia-Script-for-Windows/Build?label=GitHub%20Actions&logo=GitHub)](https://github.com/farag2/Sophia-Script-for-Windows/actions)

## Windows 11 21H2 | Windows 10 2004/20H2/21H1 | LTSC

Diff from v5.12
[5.12...5.12.1](https://github.com/farag2/Sophia-Script-for-Windows/compare/5.12...5.12.1)

<img src="https://i.imgur.com/kFHrRx1.png" height="200px"/>

As you may see, the project name changed to `Sophia Script for Windows`, due to it now supports both `Windows 10 and Windows 11` :trophy:. The project page is now <https://github.com/farag2/Sophia-Script-for-Windows>

* Fixed typo that prevets script from showing the error description;
* Fixed typos in descriptions;
* The development focus shifted to [SophiApp](https://github.com/Sophia-Community/SophiApp) :rocket:

## Sophia Script Wrapper 2.2

* Added an `Easter egg`. Find it, and let us know! 💰 

|              File                      |                          SHA-256                               |
|:--------------------------------------:|:--------------------------------------------------------------:|
| Sophia.Script.LTSC.v5.2.12.zip         |CB917D59D8E1F6105DA45B678CB4FB02A64F55CFB4DA967494D7FC3D01F1EF42|
| Sophia.Script.v5.12.1.PowerShell.7.zip   |90688BE45CE814EB485FD061BAED91B080C436020C8BC1824C5AE72B616F493B|
| Sophia.Script.v5.12.1.zip                |CFCAF9B5FF2938686FD886C13A02E6E7A954F9B53E47FCC607407F5CF4E669B7|
| Sophia Script Windows 11 v6.0.2.zip    |121122C474D4858BB89225FE2D72258A9234355A9C238B4B6A62FE119B99F2F1|
| Sophia.Script.Wrapper.v2.2.zip         |A26B66EEC81BD18D7065D9F5355C4B5EE62BD582A5B0653B3434260570A42D7C|

<hr /><em>This discussion was created from the release <a href='https://github.com/farag2/Sophia-Script-for-Windows/releases/tag/5.12.1'>06.08.2021</a>.</em>

**Comment by MLFiedler** [comment]
Hello, Farag2.  Total novice here, that is, TOTAL novice!
I'm trying to run your DeBloat script.  Have seen some very impressive details on line about it, can't wait to see how it changes the performance of my PC.
BUT:  I keep running aground when I run it in PowerShell I get the following error:
       WARNING: Windows 10 cumulative update installed: 1110. 
        The latest supported cumulative update: 1151 and higher. 
        Please run Windows Update and try again

Have  just run Win Update, and Version shows as 21H1, which I understand requires script v5.12.1.  That is what I ran to get this error.   Is there a way to FORCE windows to Update beyond the point it seems to think is the latest and greatest?

Thanks for any guidance.

Mike

**Comment by farag2** [comment]
Hi! Probably that 1151 update is optional. Do you have the "View optional updates" button in the Windows Update page?
Or install it manually: https://support.microsoft.com/en-us/topic/july-29-2021-kb5004296-os-builds-19041-1151-19042-1151-and-19043-1151-preview-6aba536a-6ed2-41cb-bc3d-3980e8693cc4

As for me now I need to disable it. Trying to firce people to use the latest Windows 10 failed. :)
I will add this check for 2 features only: the News and Interest feature and new eay to install WSL.
Also, David is cooking the Wrapper 3.0 with all new UI.

**Comment by farag2** [comment]
The direct link: http://download.windowsupdate.com/d/msdownload/update/software/updt/2021/07/windows10.0-kb5004296-x64_1d54ad8c53ce045b7ad48b0cdb05d618c06198d9.msu

**Reply by patryczek19** [reply]
Farag2 jak skompilowac sophia w visual studio chce przetlumaczyc caly sophia gui na jezyk polski krok po kroku bo jestem nowicjuszem prosze o odpowiedz

**Reply by farag2** [reply]
Hi. Thanks for your support. I do not know Polish, so had to translate your message. :cat: It's already translated: https://github.com/farag2/Sophia-Script-for-Windows/blob/master/src/Sophia_Script_for_Windows_11/Localizations/pl-PL/Sophia.psd1

SophiApp is translated too: https://github.com/Sophia-Community/SophiApp/releases/tag/1.0.92
